### ISSUE TITLE
PSI: "Browser errors were logged to console" due to Inspector-blocked resource

### ISSUE THREAD

**Issue by Krinkle** [GoogleChrome/lighthouse#10198]
URL | Description
-- | --
/stats/matomo?idsite=… | `Failed to load resource: net::ERR_BLOCKED_BY_CLIENT.Inspector`

#### What is the current behavior?

The "Best Practices" score is reduced to 92.

#### What is the expected behavior?

For resources blocked by PSI itself to not affect the score.

#### Environment Information
* Affected Channels: <https://web.dev/measure/>

**Comment by abood89721** [comment]
www.todawi.com

**Comment by ndrozd** [comment]
having. the same issue with google analytics 

`/g/collect?v=…(www.google-analytics.com) | Failed to load resource: net::ERR_BLOCKED_BY_CLIENT.Inspector`
-- | --

**Comment by Guzzox** [comment]
Any solution to fix that?

**Comment by patrickhulce** [comment]
@Krinkle @ndrozd @Guzzox any URL for us to be able to repro?

**Comment by Guzzox** [comment]
@patrickhulce My page doesn't have any Javascript in it check it out https://guzzox.xyz/

**Comment by patrickhulce** [comment]
Thanks @Guzzox! In your case it's not inspector blocked but `net::ERR_ACCESS_DENIED` on your CSS. That's strange the error appears to be logged even though the CSS loaded successfully.

@exterkamp does this help provide any clues at all?

**Comment by romainst** [comment]
Hi there,
I'm having the same issue with Facebook's Business pixel.
The request is apparently a POST which works but ends up in a 307 Internal redirect when running in a normal environment.

URL | Description
------------ | -------------
https://www.facebook.com/tr/ | Failed to load resource: net::ERR_BLOCKED_BY_CLIENT.Inspector

Example site tested: http://www.france24.com/en/
Example report with the issue : https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fwww.france24.com%2Fen%2F#best-practices

**Comment by DemkivV** [comment]
I'm having the same issue with the Matomo plugin for WordPress.

Site: https://enso-web.design/
Report: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fenso-web.design%2F
Issue: 

URL | Description
-- | --
https://enso-web.design/wp-content/plugins/matomo/app/matomo.php | Failed to load resource: net::ERR_BLOCKED_BY_CLIENT.Inspector

What's the best practice to solve this issue?

**Comment by Lofesa** [comment]
Same here whit analitycs in web.dev, not locally

![Captura](https://user-images.githubusercontent.com/22342793/97001711-213e9200-1539-11eb-9b1a-d5bf943a116d.jpg)

**Comment by spacedawwwg** [comment]
> having the same issue with google analytics
> 
> `/g/collect?v=…([www.google-analytics.com](http://www.google-analytics.com))	Failed to load resource: net::ERR_BLOCKED_BY_CLIENT.Inspector`

I came here hoping to find a solution to this. It seems to be when using Google's new analytics maybe?

**Comment by Osherz5** [comment]
Try turning off your ad blocker!

**Comment by Lofesa** [comment]
What add-blocker can be turned off in PSI or web.dev ?

**Comment by Lunnatica** [comment]
Hi,

Is there any update on this issue?

I've started seeing this error in some of my websites when using the new version of Google Analytics.

URL | Description
-- | --
/g/collect?v=…(www.google-analytics.com) | Failed to load resource: net::ERR_BLOCKED_BY_CLIENT.Inspector

Example site: https://allthemeasurements.com/
Lighthouse report with the issue: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fwww.allthemeasurements.com%2F

**Comment by crm911** [comment]
+1 for using the new Google Analytics:
Failed to load resource: net::ERR_BLOCKED_BY_CLIENT.Inspector

**Comment by gusbemacbe** [comment]
And YouTube:

![image](https://user-images.githubusercontent.com/2257024/102705861-c7252900-426a-11eb-8008-ec40fbedc4e6.png)

**Comment by tjkohli** [comment]
Any progress on this? Still getting this error with Google Analytics 4 using https://web.dev/measure

**Comment by connorjclark** [comment]
Expect a fix within a week, when we next upgrade the Lighthouse version of PSI/web.dev

**Comment by Frackher** [comment]
Hello, is there any update on this ?
Evry website using new GA tags are getting "Failed to load resource: net::ERR_BLOCKED_BY_CLIENT.Inspector" and so a 93/100.
Thanks.

**Comment by connorjclark** [comment]
This is fixed in PSI/web.dev. Are you getting a Lighthouse report from there or somewhere else? Please provide a URL for the page you are testing.

**Comment by Frackher** [comment]
@connorjclark Yes sorry, I'm using LH with pagespeed insight api. The 93% is still present on the API, maybe I've to report elsewhere ?

**Comment by Lofesa** [comment]
The issue still present in webdev.
`https://intersindicalrm.org/` with this url
![archivo](https://user-images.githubusercontent.com/22342793/109924331-67aeb500-7cc0-11eb-8514-505df066dcd7.JPG)

**Comment by paulirish** [comment]
upstream bug: b/177367218

**Comment by Frackher** [comment]
Have also this bug whil calling API PageSpeed Insights.
Does I need to specify in another thread or repo ?

**Comment by connorjclark** [comment]
It'll be fixed in the next release we do of PSI. I'd expect within 1-2 weeks.

**Comment by AdityaSaroj** [comment]
Was this ever fixed? I have the same issue with tawk.to
![image](https://user-images.githubusercontent.com/55663527/236296837-4f6054b5-2a81-41f7-859f-9f88642138fc.png)

**Comment by connorjclark** [comment]
@AdityaSaroj please open a new issue with info from the bug template filled out.
